### ISSUE TITLE
BG_VoiceParseCommand() looks for complete file path so voice system is not extension independent

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#992]
Even if the engine allows to load path without extension or with a different extension, `BG_VoiceParseCommand()` in [src/shared/bg_voice.cpp:389](https://github.com/Unvanquished/Unvanquished/blob/a6defe74d0a787d73216251c4fb95f9a4fe683b9/src/shared/bg_voice.cpp#L389) does a check on complete file path:

```c++
if ( !trap_FS_FOpenFile( token.string, nullptr, fsMode_t::FS_READ ) )
{
	int  line;
	char filename[ MAX_QPATH ];

	trap_Parse_SourceFileAndLine( handle, filename, &line );
	Log::Warn( "BG_VoiceParseCommand(): "
	            "track \"%s\" referenced on line %d of %s does not exist",
	            token.string, line, filename );
}
```

Because of that, `main/voice/default.voice` needs to enforce file extension. Other sound code unrelated to voice system (and buildable one) does not need extension.

**Comment by illwieckz** [comment]
By the way, it looks like the whole `vsay` mechanism is broken.

**Comment by illwieckz** [comment]
fixed by 6db0535022fa6f3b200b97c125bdcc1142411e94
